content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Two Common Multi-Tenancy Models For Kubernetes'
```

Comment:
Any chance we could change this to "Multi-Team" instead? This will exclude cases where one cluster runs deployments for several tenants, none of whom have access to the cluster (canonical example: multiple Wordpress sites for different users on the same cluster).

Reply:
Even if the end (app) user does not have access to the cluster, it seems to me that both tenancy models would still be applicable to a ISV / service provider and from their perspective "tenant" would be a suitable term.

Reply:
Hmm, fair enough. Maybe I'll look for individual places where we could make this distinction. Also, we're not claiming the list is exhaustive.

One last thought - "cluster-per-tenant" is arguably not a **multi** tenancy model from K8s' perspective. How about calling this article "Two Common Tenancy Models for Kubernetes"? And changing "Multitenancy" to "Tenancy" in the appropriate places throughout? But I don't feel strongly about this.